steps/identity/appellant-international-contact-details/content.cy.json

```diff
@@ -1,12 +1,21 @@
 {
   "titleHead": "Manylion cyswllt",
-  "title": "Rhowch eich manylion cyswllt",
-  "subtitle": "Fel y penodai, rhowch eich manylion cyswllt chi.",
+  "title": {
+    "withoutAppointee": "Nodwch eich manylion cyswllt",
```

Comment:
should this not be the same as the title translation above in what was line 3

Reply:
Nope, again previous content used here was actually appointee related, but there isn't a direct translation for this we can use, as it's different wording from the regular appellantContactDetails page.